contracts/test/vault/redeem.js

```diff
@@ -211,33 +204,30 @@ describe("Vault Redeem", function () {
     await setOracleTokenPriceUsd("DAI", "1.20");
     await vault.connect(governor).rebase();
 
-    // Anna's share of OUSD is 250/450
-    // Vault has 100 USDC and 350 DAI total
-    // 250/450 * (100 * 1.3 + 350 * 1.2)
-    await expect(anna).has.an.approxBalanceOf("305.55", ousd);
+    // Anna's balance does not change with the rebase
+    await expect(anna).has.an.approxBalanceOf("250.00", ousd);
 
     // Withdraw all
     await vault.connect(anna).redeemAll();
 
-    // 100 USDC and 350 DAI in contract
-    // 100 * 1.30 = 130 USD value for USDC
-    // 350 * 1.20 = 420 USD value for DAI
-    // 100/450 * 305.5555555 = 67.9012345556 USDC
-    // 350/450 * 305.5555555 = 237.654320944 DAI
-    // Difference between output value and redeem value is:
-    // 305.5555555 - (67.9012345556 * 1.3 + 237.654320944 * 1.20) = -67.9012345551
-    // Total from price is (67.9012345556 * 1.3 + 237.654320944 * 1.20) = 373.45679005508
-    // Remove 67.9012345551 * 67.9012345556 / 373.45679005508 from USDC
-    // Remove 67.9012345551 * 237.654320944 / 373.45679005508  from DAI
-    // (1000-100) + 100/450 * 305.5555555 - 67.9012345551 * 67.9012345556 / 373.45679005508 USDC
-    // (1000-150) + 350/450 * 305.5555555 - 67.9012345551 * 237.654320944 / 373.45679005508 DAI
+    // OUSD to Withdraw	250
+    // Total Vault Coins	450
+    // USDC Percentage	100	/	450	=	0.222222222222222
+    // DAI Percentage	350	/	450	=	0.777777777777778
+    // USDC Value Percentage			0.222222222222222	*	1.3	=	0.288888888888889
+    // DAI Value Percentage			0.777777777777778	*	1.2	=	0.933333333333333
+    // Output to Dollar Ratio	1.22222222222222
+    // USDC Output	250	*	0.222222222222222	/	1.22222222222222	=	45.4545454545454
+    // DAI Output	250	*	0.777777777777778	/	1.22222222222222	=	159.090909090909
+    // Expected USDC	900	+	45.4545454545454	=	945.454545454545
+    // Expected DAI	850	+	159.090909090909	=	1009.09090909091
```

Comment:
Pretty minor nit but the formatting looks off here in GitHub code review, haven't looked at in vim yet

Reply:
Lol, it's literally a spreadsheet copy/pasted. I can take out the extra spaces.

Reply:
And would you look at that, it looks fine here when I've commented on it haha

Reply:
Tabs VS spaces. :)